File: URI/2930.py
E,D = list(map(int,input().split()))

if E > D:
    print("Eu odeio a professora!")
else:
    if(E + 3 <= D):
        print("Muito bem! Apresenta antes do Natal!")
    else:
        print("Parece o trabalho do meu filho!")
        if E + 2 < 24:
            print("TCC Apresentado!")
        else:
            print("Fail! Entao eh nataaaaal!")

File: URI/C/2930.c
#include<stdio.h>

int main(){
    int E,D;
    scanf("%d %d",&E,&D);
    if(E > D)
        printf("Eu odeio a professora!\n");
    else{
        if(E + 3 <= D)
            printf("Muito bem! Apresenta antes do Natal!\n");
        else{
            printf("Parece o trabalho do meu filho!\n");
            if(E + 2 < 24)
                printf("TCC Apresentado!\n");
            else
                printf("Fail! Entao eh nataaaaal!\n");
        }

    }

    return 0;
}
